player1.gd

```diff
@@ -8,9 +8,16 @@ extends KinematicBody2D
 const GRAVITY = 350.0
 const SPEED = 200
 const AUTO_JUMP = true
+var loop_on_x = true setget set_loop_on_x, get_loop_on_x
```

Comment:
What is this for?

Reply:
That is for looping back and forth on the x.